app/templates/dashboard.html

```diff
@@ -79,16 +80,7 @@ <h1 class="govuk-heading-xl">{% trans %}Your applications{% endtrans %}</h1>
 
 {# TODO: restore this once Welsh translation is completed #}
 
-{#<p class="govuk-body">{% trans url=url_for('select_language', locale=('cy' if get_lang() == 'en' else 'en')) %} This service is also available <a href="{{ url }}" class="govuk-link">in Welsh (Cymraeg){% endtrans %}</a>.</p>#}
-{##}
-{#<div class="govuk-warning-text">#}
-{#    <span class="govuk-warning-text__icon" aria-hidden="true">!</span>#}
-{#    <strong class="govuk-warning-text__text">#}
-{#      <span class="govuk-warning-text__assistive">Warning</span>#}
-{#      {% trans %}You cannot change the language of an existing application.{% endtrans %}#}
-{#    </strong>#}
-{#  </div>#}
-
+{{ available_in_language(inset=True, warn=True )}}
```

Comment:
Rather than not merging can we comment this out and add a a TODO to restore once translation is complete? (similar to deleted block)

(This way we could get it merged and avoid possible bit-rot)

Reply:
Sure.  Good point

Reply:
I've done this in https://github.com/communitiesuk/funding-service-design-frontend/pull/220/commits/a3910f0040c554ea0b5683544f767282d15cd264